src/Components/EditProfileModal/EditProfileModal.jsx

```diff
@@ -0,0 +1,102 @@
+import { useUserContext } from "Context";
+import { useState } from "react";
+import * as styles from "./EditProfileModal.module.css";
+
+export const EditProfileModal = ({
+  firstName,
+  bio,
+  profilePhoto,
+  coverPhoto,
+  setModal,
+}) => {
+  const { editProfile } = useUserContext();
+  const [editedData, setEditedData] = useState({
+    firstName: firstName,
+    bio: bio,
+    profilePhoto: profilePhoto,
+    coverPhoto: coverPhoto,
+  });
+  // console.log(editedData);
+  return (
+    <div className={styles.edit_modal}>
+      <div className={styles.modal}>
+        <div className={styles.header}>
+          <div className={styles.left}>
+            <span onClick={() => setModal(false)} className="material-icons">
+              close
+            </span>
+            <h1>Edit profile</h1>
+          </div>
+          <button
+            onClick={() => {
+              editProfile(editedData);
+              setModal(false);
+            }}
+          >
+            Save
+          </button>
+        </div>
+        {coverPhoto ? (
+          <img className={styles.profile_backdrop} src={coverPhoto} alt="" />
+        ) : (
+          <div className={styles.no_img}></div>
+        )}
+        <div className={styles.user_img}>
+          <img
+            src={
+              profilePhoto
+                ? profilePhoto
+                : "https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png"
+            }
+            alt="Profile Photo"
+          />
+          <div className={styles.photo_upload_div}>
+            <input
+              onChange={(e) =>
+                setEditedData({
+                  ...editedData,
+                  profilePhoto: e.target.files[0],
+                })
+              }
+              className={styles.photo_change}
+              id="photo_change"
+              type="file"
+            />
+            <label htmlFor="photo_change">
+              <img
+                className={styles.label_img}
+                src="https://w7.pngwing.com/pngs/386/187/png-transparent-computer-icons-blog-change-angle-text-logo-thumbnail.png"
+                alt="change image label"
+                accept="image/jpeg,image/png,image/webp"
```

Comment:
`accept` should be in `<img />` or in `<input />`??

Reply:
Yes it should be inside input Thanks for pointing it out!